extending/custom-content-types/index.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Adding REST API Support For Custom Content Types
+---
+
+The REST API can create routes for custom post types and custom taxonomies, inside of the `wp/v2` namespace, using the same controllers as the default post type or taxonomy term controllers. Alternatively, you can use your own controllers and namespace. This document will cover using the default controllers for you custom content type's API routes. This is the easiest way and ensures the highest chance of compatibility with third parties.
```

Comment:
Small typo here:
"you custom content" I think you meant "<strong>your</strong> custom content"